cypress/e2e/marc-authority/create-new-marc-bib.cy.js

```diff
@@ -0,0 +1,87 @@
+import TestTypes from '../../support/dictionary/testTypes';
+import DevTeams from '../../support/dictionary/devTeams';
+import Permissions from '../../support/dictionary/permissions';
+import TopMenu from '../../support/fragments/topMenu';
+import Users from '../../support/fragments/users/users';
+import InventoryInstance from '../../support/fragments/inventory/inventoryInstance';
+import QuickMarcEditor from '../../support/fragments/quickMarcEditor';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import MarcAuthority from '../../support/fragments/marcAuthority/marcAuthority';
+
+describe('MARC -> MARC Bibliographic -> Create new MARC bib', () => {
+  const testData = {
+    tags: {
+        tag100: '100',
+        tag600: '600',
+        tag700: '700',
+        tag800: '800',
+        tag240: '240',
+        tag245: '245',
+        tagLDR: 'LDR',
+    },
+
+    fieldContents: {
+        tag100Content: 'Author, Person',
+        tag600Content: 'New subject',
+        tag700Content: 'Co-author, Person',
+        tag800Content: 'New series',
+        tag240Content: 'New alt. title',
+        tag245Content: 'New title',
+        tagLDRContent: '00000naa\\a2200000uu\\4500',
+    },
+
+    accordions: {
+        contributor: 'Contributor',
+        subject: 'Subject',
+        titleData: 'Title data',
+    }
+  };
+
+  let importedInstanceID = [];
+
+  before(() => {
+    cy.createTempUser([
+      Permissions.inventoryAll.gui,
+      Permissions.uiQuickMarcQuickMarcBibliographicEditorCreate.gui,
+      Permissions.uiQuickMarcQuickMarcBibliographicEditorAll.gui,
+    ]).then(createdUserProperties => {
+      testData.userProperties = createdUserProperties;
+    });
+  });
+
+  beforeEach('Login to the application', () => {
```

Comment:
do we need beforeEach hook if there's only one test in file? can we just use login inside before hook?

Reply:
yeah there will be added another tests to this file, that's why I'm using here beforeEach hook.